doc/logging.md

```diff
@@ -0,0 +1,24 @@
+# How does Happy test log work
+
+## Types of Happy logs
+  * Happy host logs — Happy will run as host, Happy host logs will have logs from Happy tool only.
```

Comment:
What do we mean by "host" here or is that introduced in other documentation? If not, we should clarify that Happy's role as "host" is that of test coordination and orchestrator, facilitating the activities of the virtual test device nodes.

Reply:
Changed "Happy host logs" to "Happy debug logs", explained Happy's role in the test too.